test/integration/auction.test.js

```diff
@@ -47,13 +50,33 @@ describe('Auctions', function () {
           expect(res.body).to.be.a('object')
           expect(res.body.message).to.be.equal('Auctions returned successfully!')
           expect(res.body.auctions).to.be.a('array')
+
+          return done()
+        })
+    })
+
+    it('Should return 404, for Auction not Found', function (done) {
+      chai
+        .request(app)
+        .get('/auction')
```

Comment:
wait, this route itself seems wrong 😅

Reply:
In thought of creating the 404 error i came up with this condition.
I'll make the appropriate changes.